show/main.py

```diff
@@ -1689,6 +1689,22 @@ def ztp(status, verbose):
        cmd = cmd + " --verbose"
     run_command(cmd, display_cmd=verbose)
 
+#
+# 'static anycast gateway' command ("show static-anycast-gateway")
+#
+@cli.command('static-anycast-gateway')
+@clicommon.pass_db
+def sag(db):
+    """Show static anycast gateway information"""
+    sag_entry = db.cfgdb.get_entry('SAG', 'GLOBAL')
```

Comment:
Can "show static-anycast-gateway" also show the list of VLANs for which static-anycast-gateway is enabled. This will eliminate the need for 2 commands to be executed to check the configuration

Reply:
Added static anycast gateway enabled VLAN interfaces in the display format.